packages/mui-material/src/Button/Button.js

```diff
@@ -215,7 +215,6 @@ const ButtonRoot = styled(ButtonBase, {
       {
         props: {
           size: 'small',
-          variant: 'text',
```

Comment:
The `text` variant is default, so in principle this shouldn't change a lot the behavior. It only rely matters when a custom variant is being created.